### ISSUE TITLE
Positive hook test at the elbow

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10814]
**Preferred term label:**
Positive hook test at the elbow

**Synonyms**

**Definition (free text, please give PubMed ID)**
With forearm fully supinated and elbow flexed to 90 degrees, examiner hooks a finger a bit around the distal biceps tendon at the anterior aspect of the elbow. Test is positive when there is lack of appreciable cord like structure to hook around.

https://www.physio-pedia.com/Elbow_Hook_Test

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Distal biceps tendon tear

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001149
- label: Positive hook test at the elbow
- parents: [Abnormal joint physiology (HP:0034430)](https://hpo.jax.org/app/browse/term/HP:0034430)
- pmid: PMID:32788933
- def: With forearm fully supinated and elbow flexed to 90 degrees, examiner hooks a finger a bit around the distal biceps tendon at the anterior aspect of the elbow. Test is positive when there is lack of appreciable cord like structure to hook around.
- comment: This sign can indicate a distal biceps tendon tear.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10814
- synonyms: Positive distal biceps hook test(NONE)|Positive O'Driscoll hook test(NONE)
